components/desktop/synergy/Makefile

```diff
@@ -9,20 +9,19 @@
 #
 
 #
-# Copyright 2015 Alexander Pyhalov
+# Copyright 2019 benf33
```

Comment:
Please, keep my Copyright and add your below. Also, copyright with nickname seems to be no-op to me (please, use full name).